Source/Basic Shapes/SvgVisualElement.cs

```diff
@@ -197,7 +230,7 @@ protected internal virtual void RenderFill(ISvgRenderer renderer)
         {
             if (this.Fill != null)
             {
-                using (var brush = this.Fill.GetBrush(this, renderer, Math.Min(Math.Max(this.FillOpacity * this.Opacity, 0), 1)))
+                using (var brush = this.Fill.GetBrush(this, renderer, Math.Min(Math.Max(this.FillOpacity, 0), 1)))

```

Comment:
I think this fixes #437 that just came in...

Reply:
I think that should be set to `fill-opacity` and not `opacity`.

Reply:
I will fix #437 after this PR.